MANIFEST.in

```diff
@@ -1,10 +1,16 @@
 include *.txt
+include *.py
```

Comment:
Are there python files that are not part of the installed package? If there aren't, this isn't necessary.